docs/src/adr/002adr-types.md

```diff
@@ -248,29 +248,24 @@ Find(Pred(_), es) ==
     ELSE -1
 ```
 
-The following definition declares a (global) recursive function, not an
+The following definition declares a (global) function, not an
 operator. However, the annotation syntax is quite similar to that of the
 operators (note though that we are using `->` instead of `=>`):
 
 ```tla
-\* @type: Set(a) -> Int;
-Card[S \in T] ==
-    IF S = {}
-    THEN 0
-    ELSE LET \* @type: a;
-             \* we could also write: "() => a" instead of just "a"
-             one_elem == (CHOOSE x \in S: TRUE)
-         IN
-         1 + Card[S \ {one_elem}]
+\* @type: (a -> b) -> Int;
+CardDomain[f \in T] ==
+    LET \* @type: Set(a);
+        \* we could also write: "() => Set(a)" instead of just "Set(a)"
+        D == DOMAIN f
+    IN LET \* @type: (Int, Int) => Int;
+           PlusOne(p,q) == p + 1
+    IN FoldSet(PlusOne, 0, D)
 ```
 
-In the definition of `Card`, we annotated the let-definition `one_elem` with its type, though any type checker should be
+In the definition of `CardDomain`, we annotated the let-definition `D` with its type, though any type checker should be
 able to compute the type of
```

Comment:
Can we keep some consistent form of line-wrapping here?

Reply:
That's hard, because Igor has some weird script that autoformats line breaks in random places. I don't want to have to manually edit those away every time.

Reply:
You don't have to manually edit. IntelliJ joins lines with Ctrl-Shift-J and [Sublime does so with Cmd-Shift-J](https://stackoverflow.com/a/15326707).

Reply:
Yes, but the next time he edits the script will insert them back. I guess the point is, we don't have a formatting style defined for `.md` (like we do for `.scala`, where there's a line-break at lines over X chars).

Reply:
Then I will complain on Igor's PR as I do on yours. @shonfeder [asked for consistent MD formatting on my recent PR](https://github.com/informalsystems/apalache/pull/1562#discussion_r836563166); I think this is a reasonable thing to do.

Reply:
Maybe the solution is to add an autoformatting step for markdown to our `fix-fmt` target.

I do like consistent formatting, but don’t think formatting issues should block prs.

Reply:
We discussed MD formatting in the standup yesterday, I think we wanted to pick it up again next week when both of you are back.
FWIW, there's plenty of other comments, the PR is not blocked on this.